ws-services/src/main/java/org/egov/waterconnection/repository/rowmapper/WaterRowMapper.java

```diff
@@ -9,109 +9,141 @@
-				currentWaterConnection.setNoOfTaps(rs.getInt("noOfTaps"));
-				currentWaterConnection.setProposedPipeSize(rs.getDouble("proposedPipeSize"));
-				currentWaterConnection.setProposedTaps(rs.getInt("proposedTaps"));
-				currentWaterConnection.setRoadCuttingArea(rs.getFloat("roadcuttingarea"));
-				currentWaterConnection.setRoadType(rs.getString("roadtype"));
-				HashMap<String, Object> additionalDetails = new HashMap<>();
-				additionalDetails.put(WCConstants.ADHOC_PENALTY, rs.getBigDecimal("adhocpenalty"));
-				additionalDetails.put(WCConstants.ADHOC_REBATE, rs.getBigDecimal("adhocrebate"));
-				additionalDetails.put(WCConstants.ADHOC_PENALTY_REASON, rs.getString("adhocpenaltyreason"));
-				additionalDetails.put(WCConstants.ADHOC_PENALTY_COMMENT, rs.getString("adhocpenaltycomment"));
-				additionalDetails.put(WCConstants.ADHOC_REBATE_REASON, rs.getString("adhocrebatereason"));
-				additionalDetails.put(WCConstants.ADHOC_REBATE_COMMENT, rs.getString("adhocrebatecomment"));
-				additionalDetails.put(WCConstants.INITIAL_METER_READING_CONST, rs.getBigDecimal("initialmeterreading"));
-				additionalDetails.put(WCConstants.APP_CREATED_DATE, rs.getBigDecimal("appCreatedDate"));
-				additionalDetails.put(WCConstants.DETAILS_PROVIDED_BY, rs.getString("detailsprovidedby"));
-				additionalDetails.put(WCConstants.ESTIMATION_FILESTORE_ID, rs.getString("estimationfileStoreId"));
-				additionalDetails.put(WCConstants.SANCTION_LETTER_FILESTORE_ID, rs.getString("sanctionfileStoreId"));
-				additionalDetails.put(WCConstants.ESTIMATION_DATE_CONST, rs.getBigDecimal("estimationLetterDate"));
-				currentWaterConnection.setAdditionalDetails(additionalDetails);
-				currentWaterConnection.processInstance(ProcessInstance.builder().action((rs.getString("action"))).build());
-				currentWaterConnection.setPropertyId(rs.getString("property_id"));
-				// Add documents id's
-				currentWaterConnection.setConnectionExecutionDate(rs.getLong("connectionExecutionDate"));
-				
-				AuditDetails auditdetails = AuditDetails.builder()
-	                        .createdBy(rs.getString("ws_createdBy"))
-	                        .createdTime(rs.getLong("ws_createdTime"))
-	                        .lastModifiedBy(rs.getString("ws_lastModifiedBy"))
-	                        .lastModifiedTime(rs.getLong("ws_lastModifiedTime"))
-	                        .build();
-				 currentWaterConnection.setAuditDetails(auditdetails);
-				 
-				connectionListMap.put(Id, currentWaterConnection);
-			}
-			addChildrenToProperty(rs, currentWaterConnection);
-		}
-		return new ArrayList<>(connectionListMap.values());
-	}
+    public List<WaterConnection> extractData(ResultSet rs) throws SQLException, DataAccessException {
+        Map<String, WaterConnection> connectionListMap = new HashMap<>();
+        WaterConnection currentWaterConnection = new WaterConnection();
+        while (rs.next()) {
+            String Id = rs.getString("connection_Id");
```

Comment:
Ideally this should be `connectionId` and not `connection_id` as we have done with other field. I assume this is a old field from previous services, so can be left as it is. But point to be noted for further development

Reply:
Ok, we will keep this thing in mind